app/components/enrolment_confirmation_component/enrolment_confirmation_component.html.erb

```diff
@@ -0,0 +1,28 @@
+<% if @current_user %>
+  <% if @programme.enrolment_confirmation_required? %>
+    <div class="enrolment-confirmation-component">
+      <%= render ModalComponent.new(
+        title: "Enrol on the #{@programme.certificate_name}",
+        reopen_button_text: (@button_text || "Enrol"),
+        show_corner_decoration: false,
+        reopen_button_class: ("enrolment-confirmation-component__button--full-width" if @full_width)
+        ) do |component| %>
+        <% component.with_body do %>
+          <div class="enrolment-confirmation-component__wrapper">
+            <div>
+              <h2 class="govuk-heading-s">Ready to enrol?</h2>
+              <p class="govuk-body">Confirm your enrolment on the certificate, and then let"s find you the perfect training course to get started with.</p>
+            </div>
+            <div>
+              <%= link_to "Comfirm my enrolment", @programme.enrol_path(user_programme_enrolment: { user_id: @current_user.id, programme_id: @programme.id }), method: :post, class: "govuk-button button", role: "button", draggable: "false" %>
+            </div>
+          </div>
+        <% end %>
+      <% end %>
+    </div>
+  <% else %>
+    <%= link_to (@button_text || "Enrol"), @programme.enrol_path(user_programme_enrolment: { user_id: @current_user.id, programme_id: @programme.id }), method: :post, class: button_classes, role: "button", draggable: "false" %>
+  <% end %>
+<% else %>
+  <%= link_to "Log in", auth_url, method: :post, class: button_classes, role: "button", draggable: "false" %>
```

Comment:
Do we need the draggable? Not sure what its needed for?

Reply:
I copied this link_to straight from one of the existing pages, now you point it out I'm not sure why draggable is in there either. I'll remove it.